modules/placement-driver/src/integrationTest/java/org/apache/ignite/internal/placementdriver/PlacementDriverManagerTest.java

```diff
@@ -247,18 +256,18 @@ public void afterEach() throws Exception {
     }
 
     private void stopPlacementDriverManager() throws Exception {
-        placementDriverManager.beforeNodeStop();
-        metaStorageManager.beforeNodeStop();
-        raftManager.beforeNodeStop();
-        clusterService.beforeNodeStop();
-        vaultManager.beforeNodeStop();
-
-        placementDriverManager.stop();
-        metaStorageManager.stop();
-        raftManager.stop();
-        anotherClusterService.stop();
-        clusterService.stop();
-        vaultManager.stop();
+        List<IgniteComponent> igniteComponents = List.of(
+                placementDriverManager,
+                metaStorageManager,
+                raftManager,
+                clusterService,
+                anotherClusterService,
+                vaultManager
+        );
+
+        IgniteUtils.closeAll(igniteComponents.stream().filter(Objects::nonNull).map(component -> component::beforeNodeStop));
+
+        IgniteUtils.stopAll(igniteComponents.stream());
```

Comment:
Same as above. Meaning exception handling.

Reply:
fix it